.github/workflows/arm-runner.yml

```diff
@@ -8,8 +8,23 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        base_image: ['raspios_lite:2021-10-30', 'dietpi:rpi_armv6_bullseye']
-
+        target: [zero_20211030, zero, zero_dietpi, zero2, zero2_64]
+        include:
+        - target: zero_20211030
+          cpu: arm1176
+          base_image: raspios_lite:2021-10-30
+        - target: zero
+          cpu: arm1176
+          base_image: raspios_lite:latest
```

Comment:
I do not quite understand the need to build both for 2021-10-30 and for latest (i.e. 2022-01-28)...

Reply:
The first build is about running tests, and we probably want to run tests on the current release. Isn't 2021-10-30 the base image of pynab 0.9?

Reply:
>The first build is about running tests,
Indeed.
>and we probably want to run tests on the current release. Isn't 2021-10-30 the base image of pynab 0.9?
No, since 0.9.1 was released in June '21, its base image was 2021-05-07.

Reply:
OK. So we should put 2021-05-07 here, this was what was intended.